using_redcap_api/using_redcap_api.md

```diff
@@ -0,0 +1,455 @@
+
+The API Playground has several parts:
+
+* The API Method selector menus
+* Raw Request Parameters
+* Response
+* Code Examples
+
+Let's look at how each part of the API Playground helps you learn the API.
+
+### API Method
+
+In your REDCap project, head over to the "API Playground" – click on that phrase in the Applications pane on the left. Then, at the top of your screen, you'll see the menu-driven selection box. The first selection you have to make is **API Method**.
+
+For now, choose "Export Records" as the API method.  This will allow you to download records (rows of data).  Make sure that:
+
+* The "Format" (second option down) is "CSV" and
+* The "Type" (third option down) is "flat".
+
+You don't have to change any other fields.  Leaving everything else alone means you'll download all the records (rows) and all of the forms / instruments with all of their fields (columns) into a single .csv file.
+
+<div class = "warning">
+Be careful not to choose `.json` as the format -- `.csv`, or **comma separated values**, is the file format you want to download!
+</div>
+
+The "Raw Request Parameters" box below your selection will change to reflect whatever you chose. Here's an example of what you might see, if you're using the cervical cancer data.  If you're using your own dataset, things will look different.  Note that we've blurred out the API key here -- that's not something we want to share!
+
+![A menu-driven way to choose API method and the coded version of that selection below.](media/api_method.png)<!-- style = "border :1px solid rgb(var(--color-highlight));" -->
+
+This "Raw Request Parameters" gives you a quick look at the information you're passing to REDCap, to make sure you selected what you really wanted in the menu part above.  Go on to the next page to see how to submit this API request!
+
+### Execute Request
+
+Look further down the API Playground page, in the "Response" area, and click on the button that says "Execute Request." You might get a "waiting" spinner, and then a box will appear below the button with the data that REDCap returned from your request.
+
+You requested your data to be in a .csv, so you should get some data that's "comma separated" – a bunch of fields that are separated by commas, with each new line of data being separated by a line break.
+
+<div class = "warning">
+
+In the "Response" box, data is presented in plain text, not in a table, so it might look confusing or overwhelming. If you want to, you can copy that plain text by clicking on the contents of the box and then using your browser's "Select All" function, followed by "Copy." Then paste it into a text editor like Atom or Notepad, saving it with the .csv extension. That will allow you to then open it in Excel to see if the .csv is what you intended. Below, here's the "before" (comma separated values in plain text) and the "after" (text saved as a .csv and then opened in Excel) of the cervical cancer data:
```

Comment:
```suggestion
In the **Response** box, data is presented in plain text, not in a table, so it might look confusing or overwhelming. If you want to, you can copy that plain text by clicking on the contents of the box and then using your browser's "Select All" function, followed by "Copy." Then paste it into a text editor like Atom or Notepad, saving it with the .csv extension. That will allow you to then open it in Excel to see if the .csv is what you intended. Below, here's the cervical cancer data before, as comma separated values in plain text, and after, as text saved as a .csv and then opened in Excel:
```